node/src/components/block_synchronizer/block_builder.rs

```diff
@@ -571,13 +571,13 @@ impl BlockBuilder {
             block_execution_results_or_chunk,
             self.should_fetch_execution_state,
         ) {
-            Ok(maybe) => {
+            Ok(RegisterExecResultsOutcome {
+                exec_results,
+                acceptance,
+            }) => {
                 debug!("register_fetched_execution_results: Ok(maybe)");
```

Comment:
Should we update the message here?

Reply:
Fixed in [60de830](https://github.com/casper-network/casper-node/pull/4039/commits/60de830640564efa0db2de1200584f2cac31bbc6).